generation/print_root_pom.sh

```diff
@@ -5,7 +5,7 @@ set -e
 GENERATION_DIR=$(dirname -- "$0");
 
 # Find all Maven modules (a directory that contains pom.xml)
-find . -mindepth 2 -maxdepth 2 -name pom.xml |sort | xargs dirname \
+find . -mindepth 2 -maxdepth 2 -name pom.xml |sort --dictionary-order | xargs dirname \
```

Comment:
`sort` is platform dependent. This '--dictionary-order' fixes it. https://github.com/googleapis/google-cloud-java/issues/8520